src/foram3d/calculators/shape_factor_calculator.ts

```diff
@@ -0,0 +1,44 @@
+/// <reference path="./calculator.ts" />
+
+module Foram3D.Calculators {
+  export class ShapeFactorCalculator extends Calculator {
+    calculate(): number {
+      var centroidsPathLength, headToTailDistance;
+
+      centroidsPathLength = this.calculateCentroidsPathLength();
```

Comment:
O ile tam wyżej jest wpiździet zmiennych i może rzeczywiście taki var rozsiany po wszystkich miejscach zbyt piękny by nie był, to tu już nie ma uzasadnienia dla pisania tego inaczej niż:
```
var centroidsPathLength = this.calculateCentroidsPathLength();
var headToTailDistance  = this.calculateDistanceBetweenHeadAndTail();
```